miio/tests/test_device.py

```diff
@@ -17,6 +17,17 @@ def test_get_properties_splitting(mocker, max_properties):
     if max_properties is None:
         max_properties = len(properties)
     assert send.call_count == math.ceil(len(properties) / max_properties)
+    assert 5 == d._protocol._timeout
```

Comment:
Please create a separate test cases for the default values, maybe simply `test_timeout_and_retry` to check both of these that use the regular timeouts.

Reply:
done